pkg/daemon/writer.go

```diff
@@ -194,11 +202,14 @@ func (nw *clusterNodeWriter) SetSSHAccessed(client corev1client.NodeInterface, l
 	return <-respChan
 }
 
-func setNodeAnnotations(client corev1client.NodeInterface, lister corev1lister.NodeLister, nodeName string, m map[string]string) (*corev1.Node, error) {
+func setNodeAnnotations(client corev1client.NodeInterface, lister corev1lister.NodeLister, nodeName string, add, remove map[string]string) (*corev1.Node, error) {
```

Comment:
Ah yes, we're starting to accumulate more merge conflicts...in this case with https://github.com/openshift/machine-config-operator/pull/3141

Not a blocker of course, just noting